k8s/configuration/secret-tm.yaml

```diff
@@ -0,0 +1,102 @@
+# All secret data should be base64 encoded before embedding them here.
+# Short strings can be encoded using, e.g.
+# echo "secret string" | base64 -w 0 > secret.string.b64
+# Files (e.g. certificates) can be encoded using, e.g.
+# cat cert.pem | base64 -w 0 > cert.pem.b64
+# then copy the contents of cert.pem.b64 (for example) below.
+# Ref: https://kubernetes.io/docs/concepts/configuration/secret/
+# Unused values can be set to ""
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cloud-manager-credentials
+  namespace: default
+type: Opaque
+data:
+  # Base64-encoded Project ID
+  # Project ID used by MongoDB deployment
+  group-id: "<b64 encoded Project ID>"
+  # Base64-encoded MongoDB Agent API Key for the group
+  agent-api-key: "<b64 encoded Agent API Key>"
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mdb-certs
+  namespace: default
+type: Opaque
+data:
+  # Base64-encoded, concatenated certificate and private key
+  mdb-instance.pem: "<b64 encoded, concatenated certificate and private key>"
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mdb-mon-certs
+  namespace: default
+type: Opaque
+data:
+  # Base64-encoded, concatenated certificate and private key
+  mdb-mon-instance.pem: "<b64 encoded, concatenated certificate and private key>"
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mdb-bak-certs
```

Comment:
The MongoDB Backup certs can be removed, no?

Reply:
yes.

Reply:
:+1: